genestack_client/files_util.py

```diff
@@ -194,6 +194,32 @@ def unlink_file(self, accession, parent):
         """
         self.invoke('unlinkFile', accession, parent)
 
+    def link_files(self, accessions, parents):
```

Comment:
I prefer to have collection of pairs
```python
[(parent1, accession1), (parent1, accession2)]
```
Or more complex structure there each first name is parent and other are childs
```python
(parent1, accession1, accession2, accession3), (parent2, accession4, accession5)
```